x/shield/types/proposal.go

```diff
@@ -89,29 +86,6 @@ func (scp ShieldClaimProposal) String() string {
 	return b.String()
 }
 
-// LockedCollateral defines the data type of locked collateral for a claim proposal.
-type LockedCollateral struct {
-	ProposalID uint64  `json:"proposal_id" yaml:"proposal_id"`
-	Amount     sdk.Int `json:"locked_coins" yaml:"locked_coins"`
-}
-
-// NewLockedCollateral returns a new LockedCollateral instance.
-func NewLockedCollateral(proposalID uint64, lockedAmt sdk.Int) LockedCollateral {
-	return LockedCollateral{
-		ProposalID: proposalID,
-		Amount:     lockedAmt,
-	}
-}
-
-// NewUnbondingDelegation returns a new UnbondingDelegation instance.
-func NewUnbondingDelegation(delAddr sdk.AccAddress, valAddr sdk.ValAddress, entry stakingTypes.UnbondingDelegationEntry) staking.UnbondingDelegation {
-	return staking.UnbondingDelegation{
-		DelegatorAddress: delAddr,
-		ValidatorAddress: valAddr,
-		Entries:          []stakingTypes.UnbondingDelegationEntry{entry},
-	}
-}
-
```

Comment:
can we delete these without any possibility of app hash change?

Reply:
These were never even used. So it should be fine, right?

Reply:
yes then it should be fine 🧐